pkgs/clickable-apis/index.ts

```diff
@@ -13,11 +13,69 @@ import {
   createNodePayloadClient,
 } from 'simple-payload'
 
+/**
+ * Interface for the API header object that appears at the top of all JSON responses
+ */
+export interface ApiHeader {
+  /** Name of the API */
+  name: string
+  /** Description of the API */
+  description: string
+  /** Home URL */
+  home: string
+  /** Login URL - only present when user is not logged in */
+  login?: string
+  /** Signup URL - only present when user is not logged in */
+  signup?: string
+  /** Admin URL */
+  admin: string
+  /** Documentation URL */
+  docs: string
+  /** Repository URL */
+  repo: string
+  /** SDK URL */
+  sdk: string
+  /** Site URL */
+  site: string
+  /** Discord community URL */
+  discord?: string
+  /** GitHub issues URL */
+  issues?: string
+  /** With URL - reference to apis.do */
+  with?: string
+  /** From URL - reference to agi.do */
+  from?: string
+}
+
 export type { ApiContext, PayloadClientResult, PayloadClientFn }
 
 let _currentRequest: NextRequest | null = null
 let _currentContext: ApiContext | null = null
 
+const domainDescriptions: Record<string, string> = {
```

Comment:
these descriptions should come from `api.config.ts` in the root of the project ... since that is outside the sdk package folder, if that's a problem, how can we solve it?